packages/fast-usdc-contract/src/fast-usdc.contract.ts

```diff
@@ -193,9 +193,33 @@ export const contract = async (
   });
 
   const zoeTools = makeZoeTools(zcf, vowTools);
+  const { advanceFunds } = orchestrateAll(
+    // @ts-expect-error flow membrance type debt
+    { advanceFunds: flows.advanceFunds },
+    {
+      chainHub, // XXX excess authority?
```

Comment:
please attenuate like the `forwardFunds` flow does

Reply:
Now we're getting to the kind of attenuation for orchestration flows. I would prefer that we provide an attenuated durable object here instead of a record containing a function (which is what happens inside the forwardFunds logic I believe).

Reply:
> Why? ChainHub is not durable nor was it meant to be.

In that case let's name it `chainHubTools` and have a single helper `resolveAccountId` or whatever attenuated power is necessary. I want to be clear here that the expected endowments are helper functions.